types/just-extend/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for just-extend 1.1
+// Project: https://github.com/angus-c/just#readme
+// Definitions by: Peter Safranek <https://github.com/pe8ter>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare function extend(obj1: object, ...objn: any[]): object;
+
+declare function extend(deep: boolean, obj1: object, ...objn: any[]): object;
+
+declare namespace extend {}
```

Comment:
This is an inappropriate hack. [Please read more here from the DefinitelyTyped REAMDE](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

Additionally, test files should ideally use the `import Foo = require("foo")` style syntax.

Reply:
CC @RyanCavanaugh